docs/low-code-jobs/databricks-jobs.md

```diff
@@ -3,23 +3,165 @@ sidebar_position: 1
+| Alerts Email for Pipeline | Comma separated list of emails, that are going to receive notifications on specific job status events (job start, failure, or success) for entire Pipeline.                                                                                         |
+| Per Gem Timeout           | Timeout for each Gem in job Pipeline                                                                                                                                                                                                                |
+| Number of retries per Gem | Number of retries for each Gem in job Pipeline                                                                                                                                                                                                      |
+| Alerts Email per Gem      | Comma separated list of emails, that are going to receive notifications on specific job status events (job start, failure, or success) for each Gem in job Pipeline.                                                                                |
+
 ## Deployment
 
+To deploy a job on Databricks, we need to release the project from Prophecy UI as shown in example below. As soon as the project is
+released, the job would start appearing on Databricks jobs page as well.
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://user-images.githubusercontent.com/103921419/184726064-67e3df01-ba4c-431e-92e9-8bda92a19530.mp4" title="Job Deployment" allow="autoplay;fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+
+---
+
+:::info
+
+Please ensure to enable the job before release on the job page for it to run on the specified schedule. If it is not enabled, then the job would get released but with paused state.
+
+If fabric is changed in the job, then it would create a new job in the Databricks jobs page, but the job with older fabric would be paused automatically and the new job would be scheduled.
+:::
+
+As per the size, cost and parallelism required for jobs, below different cluster modes can be used for deployment.
+
+### Multi Job Cluster Mode
+
+In this mode, each component of job would spawn a separate cluster of its own.
+
+![Multi Job Cluster](img/databricks-jobs-multi-cluster-eg.png)
+
+### Single Job Cluster Mode
+
+In this mode, each component of job would run on the same cluster.
+
+:::info
+To use single cluster mode, package name across each Pipeline in job should be unique.
+This is done to ensure that the folder structure for Pipeline does not overwrite one another.
+Please refer below example, for steps on how to configure package name in Pipeline.
```

Comment:
Phrasing: Please refer to the example below for steps on ...

Reply:
Some nits:

From the video below it seems you are walking through the Gems within the job "Python_Job_demo." (Pipeline_0, Pipeline_1, and Script_1). I am confused because Pipelines "rest_api_demo" and "test_pipeline" do not appear in the Job.  Also, what is happening at 47 seconds? A screen refresh or a particular button?

Reply:
<img width="1364" alt="image" src="https://user-images.githubusercontent.com/103921419/185066080-4e95c313-a1e4-438f-aeb6-8fb27342e428.png">

pipeline is a gem inside job, and `rest-api-demo` and `test-pipeline` are workflows configured inside that gem